cpp/ql/test/library-tests/ir/ssa/aliased_ssa_ir.expected

```diff
@@ -328,7 +328,7 @@ ssa.cpp:
 #   97|     m0_14(unknown)        = ^CallSideEffect              : ~m0_5
 #   97|     m0_15(unknown)        = Chi                          : total:m0_5, partial:m0_14
 #   97|     v0_16(void)           = ^IndirectReadSideEffect[0]   : &:r0_12, ~m0_15
-#   97|     m0_17(unknown)        = ^BufferMayWriteSideEffect[0] : &:r0_12, ~m0_15
+#   97|     m0_17(unknown)        = ^BufferMayWriteSideEffect[0] : &:r0_12
 #   97|     m0_18(unknown)        = Chi                          : total:m0_15, partial:m0_17
```

Comment:
This change LGTM. There's already a `Chi` node that explains how the old value is combined with the new value, so there should be no need for the side effect node to take the old value as argument.